pkg/Microsoft.ML.DnnImageFeaturizer.AlexNet/Microsoft.ML.DnnImageFeaturizer.AlexNet.nupkgproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.ML.OnnxTransform\Microsoft.ML.OnnxTransform.nupkgproj" />
+    <ProjectReference Include="..\Microsoft.ML.OnnxTransformer\Microsoft.ML.OnnxTransformer.nupkgproj" />
```

Comment:
What is going on here?

Reply:
Uggh, I think nuget just skips over project references it can't find.

Can we get this fix in a separate PR?

---
In reply to: [259437310](https://github.com/dotnet/machinelearning/pull/2690#discussion_r259437310) [](ancestors = 259437310)

Reply:
Correct - you get just a warning. Sure I can move this into a different PR.